test/ParseableInterface/fixed-layout-property-initializers.swift

```diff
@@ -1,65 +1,68 @@
 
-// CHECK: @_fixed_layout public struct MyStruct {
+// COMMON: @_fixed_layout public struct MyStruct {
 @_fixed_layout
 public struct MyStruct {
-  // CHECK: public var publicVar: [[BOOL:(Swift\.)?Bool]] = false
+  // COMMON: public var publicVar: [[BOOL:(Swift\.)?Bool]] = false
   public var publicVar: Bool = false
 
-  // CHECK: internal var internalVar: ([[BOOL]], [[BOOL]]) = (false, true)
+  // COMMON: internal var internalVar: ([[BOOL]], [[BOOL]]) = (false, true)
   internal var internalVar: (Bool, Bool) = (false, true)
 
-  // CHECK: private var privateVar: [[BOOL]] = Bool(4 < 10)
+  // COMMON: private var privateVar: [[BOOL]] = Bool(4 < 10)
   private var privateVar: Bool = Bool(4 < 10)
 
-  // CHECK: @usableFromInline
-  // CHECK-NEXT: internal var ufiVar: [[BOOL]] = true
+  // COMMON: @usableFromInline
+  // COMMON-NEXT: internal var ufiVar: [[BOOL]] = true
   @usableFromInline internal var ufiVar: Bool = true
 
-  // CHECK: public var multiVar1: [[BOOL]] = Bool(false), (multiVar2, multiVar3): ([[BOOL]], [[BOOL]]) = (true, 3 == 0)
+  // COMMON: public var multiVar1: [[BOOL]] = Bool(false), (multiVar2, multiVar3): ([[BOOL]], [[BOOL]]) = (true, 3 == 0)
   public var multiVar1: Bool = Bool(false), (multiVar2, multiVar3): (Bool, Bool) = (true, 3 == 0)
 
   // CHECK: @_hasInitialValue public static var staticVar: [[BOOL]]
+  // RESILIENT: public static var staticVar: [[BOOL]]
   public static var staticVar: Bool = Bool(true && false)
 
-  // CHECK: @inlinable internal init() {}
+  // COMMON: @inlinable internal init() {}
   @inlinable init() {}
 }
 
-// CHECK: @_fixed_layout public class MyClass {
+// COMMON: @_fixed_layout public class MyClass {
 @_fixed_layout
 public class MyClass {
-  // CHECK: public var publicVar: [[BOOL]] = false
+  // COMMON: public var publicVar: [[BOOL]] = false
   public var publicVar: Bool = false
 
-  // CHECK: internal var internalVar: [[BOOL]] = false
+  // COMMON: internal var internalVar: [[BOOL]] = false
   internal var internalVar: Bool = false
 
-  // CHECK: private var privateVar: {{(Swift\.)?}}UInt8 = UInt8(2)
+  // COMMON: private var privateVar: {{(Swift\.)?}}UInt8 = UInt8(2)
   private var privateVar: UInt8 = UInt8(2)
 
-  // CHECK: @usableFromInline
-  // CHECK-NEXT: internal var ufiVar: [[BOOL]] = true
+  // COMMON: @usableFromInline
+  // COMMON-NEXT: internal var ufiVar: [[BOOL]] = true
   @usableFromInline internal var ufiVar: Bool = true
 
   // CHECK: @_hasInitialValue public static var staticVar: [[BOOL]]
+  // RESILIENT: public static var staticVar: [[BOOL]]
   public static var staticVar: Bool = Bool(true && false)
 
-  // CHECK: @inlinable internal init() {}
+  // COMMON: @inlinable internal init() {}
   @inlinable init() {}
 }
 
 // CHECK: @_hasInitialValue public var topLevelVar: [[BOOL]]
+// RESILIENT: public var topLevelVar: [[BOOL]]
```

Comment:
Strictly speaking this will match both forms…you need a `{{^}}` to actually check for the attribute's absence.